pyerrors/obs.py

```diff
@@ -1550,6 +1594,41 @@ def import_jackknife(jacks, name, idl=None):
     return new_obs
 
 
+def import_bootstrap(boots, name, shape, random_numbers=None):
```

Comment:
In principle, one could allow to pass an `idl` here that is used when the Obs is constructed.